src/core/draw_tools/map-daw-tools/customDrawModes/CustomModifyMode.ts

```diff
@@ -77,7 +78,7 @@ export class CustomModifyMode extends GeoJsonEditMode {
           this._submodesCache[submodeName] = new ModifyMode();
           break;
         case 'Transform':
-          this._submodesCache[submodeName] = new TransformMode();
+          this._submodesCache[submodeName] = new CustomTransformMode();
```

Comment:
Currently the only  way to understand difference between TransformMode and CustomTransformMode in line by line soruce code comparing. Need more details